README.md

```diff
@@ -147,6 +147,10 @@ Here `run_1` to `run_n` correspond to the number of independent runs in a given
 >
 > Due to the underlying statistical aggregation relying on `numpy` array operations it is required that all data contain the same number of data points. This implies that, for a given environment, it is required that all experiment trials should be done using the same algorithms, on the same tasks, for the same number of independent runs and for the same amount of evaluation steps. The code will currently check that these conditions are met and will not be able to progress otherwise. In the case that this happens, the `check_data` method of the [`DiagnoseData`](marl_eval/utils/diagnose_data_errors.py) class will be able to tell a user exactly what is causing the issues in their raw experiment data.
 
+> 🚧 **Important note on algorithm names** 🚧
+>
+> For producing probability of improvement plots, it is important that any algorithm names in the dataset do not contain any commas.
```

Comment:
It is strange to find an algorithm name with commas :sweat_smile:, maybe we mention here that it doesn't support any punctuations in the name only "_" and "-", what do you think?

Reply:
That is very true and I don't really ever see it happening. But the code will work fine with any other punctuation marks in the algorithm names which is why I added this very particular note :sweat_smile: